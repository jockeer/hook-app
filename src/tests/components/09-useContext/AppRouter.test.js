import React from 'react';

import {mount} from 'enzyme'
import { AppRouter } from '../../../componets/09-useContext/AppRouter';
import { UserContext } from '../../../componets/09-useContext/UserContext';


describe('Pruebas en AppRouter', () => { 

    const user = {
        id:123,
        name:'Daniel'
    }

    const wrapper = mount(
        <UserContext.Provider value={{user}}>
            <AppRouter /> 

        </UserContext.Provider>
    )

    test('Debe mostrarse correctamente', () => { 
        expect(wrapper).toMatchSnapshot()
    })
})