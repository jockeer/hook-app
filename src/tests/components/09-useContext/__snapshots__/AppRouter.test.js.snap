// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en AppRouter Debe mostrarse correctamente 1`] = `
Array [
  <nav
    className="navbar navbar-expand-lg navbar-dark bg-dark"
  >
    <div
      className="container-fluid"
    >
      <a
        className="navbar-brand"
        href="/"
        onClick={[Function]}
      >
        useContext
      </a>
      <button
        aria-controls="navbarNavAltMarkup"
        aria-expanded="false"
        aria-label="Toggle navigation"
        className="navbar-toggler"
        data-bs-target="#navbarNavAltMarkup"
        data-bs-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse"
        id="navbarNavAltMarkup"
      >
        <div
          className="navbar-nav"
        >
          <a
            className="nav-link"
            href="/home"
            onClick={[Function]}
          >
            Home
          </a>
          <a
            className="nav-link"
            href="/about"
            onClick={[Function]}
          >
            About
          </a>
          <a
            className="nav-link"
            href="/login"
            onClick={[Function]}
          >
            Login
          </a>
        </div>
      </div>
    </div>
  </nav>,
  <div
    className="container"
  >
    <div>
      <h1>
        HomeScreen
      </h1>
      <hr />
      <pre>
        {
    "id": 123,
    "name": "Daniel"
}
      </pre>
    </div>
  </div>,
]
`;
